druid.go

```diff
@@ -83,6 +92,13 @@ func NewClient(baseURL string, options ...ClientOption) (*Client, error) {
 	return c, nil
 }
 
+func InsecureSkipVerify(opts *clientOptions) {
+	tlsConfig := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+	opts.httpClient.Transport = &http.Transport{TLSClientConfig: tlsConfig}
```

Comment:
Why not doing it directly from https://github.com/grafadruid/go-druid/pull/38/files#diff-7c057e108ee542c25bb8493b0f93d2be371ce382afab8d0ec50109bb4b19d492R206 since it does change the client option after all.

Reply:
Ok, got it. Let me try with `http.HTTPClient.Transport.TLSClientConfig.InsecureSkipVerify`

Reply:
`opts.httpClient.Transport` doesn't have a field by that name. It gives an error `opts.httpClient.Transport.TLSClientConfig undefined (type http.RoundTripper has no field or method TLSClientConfig)`

Reply:
You need a type assertion since `Transport` is an interface (`RoundTripper`) which doesn't have that field but the used `RoundTripper` (ak `http.Transport`) does have that field. If you need to access that field then you should do something like `http.HTTPClient.Transport.(*http.Transport).TLSClientConfig.InsecureSkipVerify`

Reply:
Got this to work, now I see what you meant. The code looks much cleaner now. Thanks.